sql/migrations/20220925023354_world.sql

```diff
@@ -0,0 +1,164 @@
+UPDATE `creature` SET `patch_min`='4' WHERE  `guid`=87044;
+-- Use Dun Morogh, Kharanos GY for Ironforge pre patch 1.6
+UPDATE `game_graveyard_zone` SET `faction`='0' WHERE  `id`=101 AND `ghost_zone`=1537;
+
+-- Set Tirisfal Glades, Ruins of Lordaeron GY to patch 1.6
+UPDATE `game_graveyard_zone` SET `patch_min`='4' WHERE  `id`=853;
+-- Set Tirisfal Glades, Ruins of Lordaeron GY Spirit Healer to patch 1.6
+UPDATE `creature` SET `patch_min`='4' WHERE  `guid`=2065;
+-- Use Tirisfal Glades, Brill GY for Undercity pre patch 1.6
+INSERT INTO `game_graveyard_zone` (`id`, `ghost_zone`, `faction`, `patch_min`) VALUES (289, 1497, 0, 0);
+
+-- Set Western Plaguelands, Caer Darrow GY to patch 1.6
+UPDATE `game_graveyard_zone` SET `patch_min`='4' WHERE  `id`=869;
+-- Set Western Plaguelands, Caer Darrow GY Spirit Healer to patch 1.6
+UPDATE `creature` SET `patch_min`='4' WHERE  `guid`=40544;
+-- Delete invalid graveyards for Caer Darrow
+DELETE FROM `game_graveyard_zone` WHERE  `id`=429 AND `ghost_zone`=2057;
+DELETE FROM `game_graveyard_zone` WHERE  `id`=629 AND `ghost_zone`=2057;
+-- Set correct graveyards for Caer Darrow pre patch 1.6
+INSERT INTO `game_graveyard_zone` (`id`, `ghost_zone`, `faction`, `patch_min`) VALUES (569, 2057, 67, 0);
+INSERT INTO `game_graveyard_zone` (`id`, `ghost_zone`, `faction`, `patch_min`) VALUES (509, 2057, 469, 0);
+
+-- Set Eastern Plaguelands, Blackwood Lake GY to patch 1.6
+UPDATE `game_graveyard_zone` SET `patch_min`='4' WHERE  `id`=909;
+-- Set Eastern Plaguelands, Blackwood Lake GY Spirit Healer to patch 1.6
+UPDATE `creature` SET `patch_min`='4' WHERE  `guid`=40551;
+-- Use Eastern Plaguelands, Light's Hope Chapel GY for Stratholm pre patch 1.6
+INSERT INTO `game_graveyard_zone` (`id`, `ghost_zone`) VALUES ('510', '2017');
+
+-- Set Feralas, Dire Maul GY to patch 1.6
+UPDATE `game_graveyard_zone` SET `patch_min`='4' WHERE  `id`=849;
+-- Set Feralas, Dire Maul GY Spirit Healer to patch 1.6
+UPDATE `creature` SET `patch_min`='4' WHERE  `guid`=40561;
+-- Use Feralas, Camp Mojache for Dire Maul GY pre patch 1.6
+INSERT INTO `game_graveyard_zone` (`id`, `ghost_zone`) VALUES ('310', '2557');
+
+-- Set The Hinterlands, The Overlook Cliffs GY to patch 1.5
+UPDATE `game_graveyard_zone` SET `patch_min`='3' WHERE  `id`=789 AND `ghost_zone`=47;
+
+-- Set Elwynn Forest, Eastvale Logging Camp GY to patch 1.6
+UPDATE `game_graveyard_zone` SET `patch_min`='4' WHERE  `id`=854;
+-- Set Elwynn Forest, Eastvale Logging Camp GY Spirit Healer to patch 1.6
+UPDATE `creature` SET `patch_min`='4' WHERE  `guid`=17650;
+
+-- Set min patch for Badlands, Graveyard NE GY
+UPDATE `game_graveyard_zone` SET `patch_min`='10' WHERE  `id`=8;
+-- Add Badlands and Uldaman to Loch Modan, Thelsamar GY
+INSERT INTO `game_graveyard_zone` (`id`, `ghost_zone`, `faction`, `patch_min`) VALUES ('6', '3', '0', '0');
+INSERT INTO `game_graveyard_zone` (`id`, `ghost_zone`, `faction`, `patch_min`) VALUES ('6', '1337', '0', '0');
```

Comment:
While Thelsamar makes a solid amount of sense as a graveyard for Uldaman berfore patch 1.12 (Similar to Twilight Vale, Darkshore for BFD), I seriously doubt it as a graveyard for the Badlands as a zone. It would be a very strange behaviour for it to be that way, since in every case I can think of if a zone has a graveyard it will not use graveyards from outside the zone for itself, and Badlands has a graveyard in Kargath.

Reply:
I've finally evidence where uldaman should have it graveyard in pre 1.12.0.
This video https://youtu.be/eHCZPBF3gAo?t=33 shows that the graveyard for Uldaman pre 1.12.0 should be in Kargath.

A screenshot from the video:
![firefox_BLMcwx6slY](https://user-images.githubusercontent.com/6137576/226173480-afe24d93-5b86-4bb9-b64b-0104820ac899.png)
A screenshot from me on 1.12.1 after releasing spirit in uldaman:
![WoW_B8H7t13HPr](https://user-images.githubusercontent.com/6137576/226173493-da4e7ee8-7c54-49e7-9021-394086c78759.png)

It confirms that the video is a ghost run to the portal in pre 1.12.0.

The SQL will be updated to the correct graveyard later today.